### PR TITLE
successIcon: allow to pass content

### PR DESCRIPTION
**Pull request by jrcastro2**
* closes https://github.com/inveniosoftware/invenio-app-rdm/issues/2076

### CODE REVIEW

File: invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/SuccessIcon.js [hunk 1]
```diff
@@ -34,19 +34,28 @@ export class SuccessIcon extends Component {
...
   render() {
-    const { className } = this.props;
+    const { className, content } = this.props;
     const { show } = this.state;
-    return show && <Icon name="checkmark" className={`positive ${className}`} />;
+    return (
+      show && (
+        <>
+          <Icon name="checkmark" className={`positive ${className}`} />
+          <span>{content && content}</span>
```
Comment (ntarocco) [inline invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/SuccessIcon.js:43]: Pseudocode:
```suggestion
          {content && <span> {content}</span>}
```

**Review by ntarocco — COMMENTED** [review]
Can you also please fix the linting issues?

File: invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/SuccessIcon.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ntarocco) [reply invenio_communities/assets/semantic-ui/js/invenio_communities/members/components/SuccessIcon.js:43]: Let's don't use `lodash` when it is not needed: if only check for `null` (default prop value), `isEmpty` is not needed.

**Review by ntarocco — APPROVED** [review]

File: invenio_communities/communities/services/service.py [hunk 2]
```diff
@@ -111,20 +111,25 @@ def delete(self, identity, id_, revision_id=None):
         return super().delete(identity, record.id, revision_id)
 
     def search_user_communities(
-        self, identity, params=None, search_preference=None, **kwargs
+        self, identity, params=None, search_preference=None, extra_filter=None, **kwargs
```
Comment (kpsherva) [inline invenio_communities/communities/services/service.py:114]: @jrcastro2 added for the service to be reusable with extra filters

**Pull request merged**